doc/installing.rst

```diff
@@ -6,11 +6,160 @@ Installing PYPIT
+
+	conda install PyQT
+
+If any of the packages are out of date, they can be updated with a command like::
+
+	conda update scipy
+
+Installing Linetools
+--------------------
+The latest version of `Linetools <https://github.com/linetools/linetools/>`_ is also required for PYPIT. 
+Linetools is a package designed for the analysis of 1-D spectra. The installation steps for linetools are provided below but much more detailed documentation is provided `here <http://linetools.readthedocs.io/en/latest/install.html/>`_. 
+
+In addition to the python packages mentioned above, linetools requires the installation of `specutils <https://github.com/astropy/specutils/>`_. Specutils can either be cloned directly from github or installed via pip.
+
+To install specutils with pip, use the command::
+
+	pip install --no-deps specutils
+
+Linetools can be installed from github or using pip. 
+To install a development version of linetools from the source::
+
+	mkdir linetools
+	cd linetools
+	git clone https://github.com/linetools/linetools.git
+	python setup.py develop
+
+To install linetools using pip::
+
+	pip install linetools
+
+According to the linetools documentation page, "If you wish to have full functionality of the GUIs and are using MacOSX, then you probably need to change your backend from macosx to TkAgg in the matplotlibrc file."
+
+If you'd like to make sure that linetools has been properly installed, you can run the following test::
+
+	python -c 'import linetools; linetools.test()'
+
+If the test reports no errors, then linetools was installed without issue. 
+
+GSL
+---
+
+The package complies Cython code that links to gsl routines.
+These must be installed on your system prior to PYPIT installation.
+We recommend that if you need to install GSL that you use Anaconda,
+e.g.::
+
+    conda install -c https://conda.anaconda.org/asmeurer gsl
+
+Beware:  multiple installations of libgsl is likely to cause a problem..
+
+You are also required to point the ENVIRONMENTAL varible
+GSL_PATH to the path above the lib/ and include/ files.
+
+You should be able to determine that path with::
+
+    gsl-config --prefix
+
+In my case, anaconda installs packages in /Users/USERNAME/anaconda 
+
+You then need to edit your .bashrc, .tcshrc, or .cshrc file with this path.
+If you are using bash (and the path mentioned above as an example)::
+
+	export GSL_PATH='/Users/USERNAME/anaconda'
+
+If instead you are using tcsh or csh::
+
+	setenv GSL_PATH "/Users/USERNAME/anaconda"
+
+Installing PYPIT
+================
+
+We recommend that you grab the code from github::
+
+    mkdir PYPIT
+	cd PYPIT
+	git clone https://github.com/PYPIT/PYPIT.git
+
+From there, you can build and install either with install or develop, e.g.::
+
+    python setup.py develop
+
+or::
+
+	python setup.py install
+
+This should compile all the necessary Cython files, etc.
+
+**If you will be editing/contributing to the code:** 
```

Comment:
This is useful for the unintiated, but probably should be removed.
We don't want novices editing PYPIT anyhow! ;)